src/anomalib/deploy/inferencers/openvino_inferencer.py

```diff
@@ -54,6 +54,8 @@
 from typing import Any
 
 import numpy as np
+import PIL.Image
```

Comment:
If we are using this just for type checking, maybe we could have something like this?
`from PIL.Image import Image as PILImage`

Then the annotation would become something like this
`def predict(self, image: str | Path | np.ndarray | PILImage | torch.Tensor) -> ImageBatch:`

Reply:
Ok